baseline/tf/classify/model.py

```diff
@@ -360,6 +361,125 @@ def embed(self, **kwargs):
         return EmbeddingsStack(self.embeddings)
 
 
+class MultiLabelMixin(ClassifierModelBase):
```

Comment:
The only difference between this and the basic classifier is the output layer, isnt there a way to avoid redefining the entire structure?  Maybe instead of using mixins we should assume that its possible for all classifiers to be sigmoid output instead of softmax and have conditions for both paths.  That is more like what I did in my branch, and while I like what you have in the PyTorch, this seems a bit sketchier

Reply:
The main driving change here was that the y placeholder is now `tf.compat.v1.placeholder(tf.float32, [None, nc], name="y")` instead of a `float32` instead of an `int32`. I suppose this class could pass in this as a value for `y` in the kwargs to avoid having to duplicate this function?